packages/react/src/components/iconCard/IconCard.tsx

```diff
@@ -73,6 +82,8 @@ export const IconCard: React.FunctionComponent<
     tooltip,
     choices,
     animateOnHover,
+    placeBadgesAbove,

```

Comment:
Will we have a time where the badges will be at the bottom of the title?

Reply:
by default they're on the right side of the content as the container is display: flex with standard horizontal direction.
I would be surprised if UX implemented a variant with the badges below as it would be so similar to having them above and would just be annoying for us. 😛 
Am I answering the question?

Reply:
ahh yes okay that makes sense, 👍 I forgot we were also on top of the `IconCard` for this that utilizes the badges on the left